examples/thermodynamics/the_boiling_time_of_the_mass_of_water.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+from sympy import solve, Symbol, Eq, dsolve
+from symplyphysics import print_expression, Quantity, prefixes, units, convert_to
+from symplyphysics.core.symbols.celsius import to_kelvin_quantity, Celsius
+from symplyphysics.laws.electricity import power_factor_from_active_and_full_power as efficiency_law
+from symplyphysics.laws.thermodynamics import energy_from_combustion as combustion_energy_law
+from symplyphysics.laws.thermodynamics import thermal_energy_from_mass_and_temperature as energy_heating_law
+from symplyphysics.laws.thermodynamics import energy_to_vaporization_from_mass as energy_to_vapor_law
+from symplyphysics.definitions import mass_flow_rate as mass_rate_law
+
+# Example from https://easyfizika.ru/zadachi/termodinamika/na-zazhzhennuyu-spirtovku-s-kpd-60-postavili-sosud-s-500-g-vody-pri-20-c-cherez-kakoe/
+# A vessel with 500 g of water at 20 ° C was placed on a lighted alcohol lamp
+# with an efficiency of 60%. After what time will 20 g of water boil off if 4 g of alcohol
+# burns in a spirit lamp in a minute?
+
+efficiency = Symbol("efficiency")
+mass_of_booling_water = Symbol("mass_of_booling_water")
+mass_of_water = Symbol("mass_of_water")
+temperature_of_water = Symbol("temperature_of_water")
+mass_of_alcohol = Symbol("mass_of_alcohol")
+time_of_minute = Symbol("time_of_minute")
+
+mass_rate_alcohol = Symbol("mass_rate_alcohol")
+
+specific_heat_of_combustion_alcohol = Symbol("specific_heat_of_combustion_alcohol")
+specific_heat_of_heating_water = Symbol("specific_heat_of_heating_water")
+specific_heat_of_vaporization_water = Symbol("specific_heat_of_vaporization_water")
+temperature_of_vaporization_water = Symbol("temperature_of_vaporization_water")
+
+time = Symbol("time")
+
+mass_gas_integral = dsolve(mass_rate_law.definition, mass_rate_law.mass(mass_rate_law.time))
+mass_of_gas_equation = mass_gas_integral.subs({
+    "C1": 0,
```

Comment:
Explain why C1 is zero.
Use temporary const value to convert integral to linear function.